app/src/main/java/org/wikipedia/games/onthisday/OnThisDayGameViewModel.kt

```diff
@@ -236,6 +247,8 @@ class OnThisDayGameViewModel(savedStateHandle: SavedStateHandle) : ViewModel() {
         val langToState = totalState.langToState.toMutableMap()
         langToState[wikiSite.languageCode] = currentState
         Prefs.otdGameState = JsonUtil.encodeToString(TotalGameState(langToState)).orEmpty()
+        savedStateHandle[KEY_CURRENT_STATE] = JsonUtil.encodeToString(currentState)
```

Comment:
The entire game state gets saved to Prefs (on the previous line). Do we really need to duplicate it into savedStateHandle?